Sources/PipeableSDK/PipeablePage.swift

```diff
@@ -203,7 +165,17 @@ public class PipeablePage {
         }
     }
 
-    public func goto(_ url: String, waitUntil: WaitUntilOption = .load, timeout: Int = 30000) async throws {
+    /// Navigates to the given URL.
+    /// - Parameters:
+    ///  - url: The URL to navigate to.
+    ///  - waitUntil: When to consider navigation as finished, defaults to `.load`.
+    ///  - timeout: Maximum time to wait for the navigation to finish, defaults to 30000ms.
+    ///  - Returns: The response of the navigation - HTTP status code and
+    ///  headers. Can return nil if the response is not available. For example,
+    ///  if the navigation is to a non-HTTP URL: about:blank, data:, etc.
+    ///  - Throws: PipeableError.navigationError if the navigation fails.
+
```

Comment:
extra line? ?!?

Reply:
What's with the exclamations? :)

Reply:
I was wondering if the line is really extra... I was wondering if that is really "extra" or this is how it should be. Sorry, for the ambiguous comment ;)